manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -409,6 +409,58 @@ data:
     # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="console_url"}'
     #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_requests_total:sum gives the total number of login requests initiated from the web console.
+    #
+    - '{__name__="cluster:console_auth_login_requests_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_successes_total:sum gives the total number of successful logins initiated from the web console.
+    # Labels:
+    # * `role`, one of `kubeadmin`, `cluster-admin` or `developer`. The value is based whether or not the logged-in user can list all namespaces.
+    #
+    - '{__name__="cluster:console_auth_login_successes_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_failures_total:sum gives the total number of login failures initiated from the web console.
+    # This might include canceled OAuth logins depenending on the user OAuth provider/configuration.
+    # Labels:
+    # * `reason`, currently always `unknown`
+    #
+    - '{__name__="cluster:console_auth_login_failures_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_logout_requests_total:sum gives the total number of logout requests send from the web console.
+    # Labels:
+    # * `reason`, currently always `unknown`
+    #
+    - '{__name__="cluster:console_auth_logout_requests_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_usage_users:max contains the number of web console users splitten into the roles.
+    # Labels:
+    # * `role`: `kubeadmin`, `cluster-admin` or `developer`. The value is based whether or not the user can list all namespaces.
+    #
+    - '{__name__="cluster:console_usage_users:max"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_plugins_info:max reports information about the web console plugins and their state.
```

Comment:
to be sure: we can have all label combinations for the same cluster? E.g. we can have

```
cluster:console_plugins_info:max{name="redhat",state="enabled"} 1
cluster:console_plugins_info:max{name="redhat",state="disabled"} 2
cluster:console_plugins_info:max{name="redhat",state="notfound"} 3
cluster:console_plugins_info:max{name="demo",state="enabled"} 4
cluster:console_plugins_info:max{name="demo",state="disabled"} 5
cluster:console_plugins_info:max{name="demo",state="notfound"} 6
cluster:console_plugins_info:max{name="other",state="enabled"} 7
cluster:console_plugins_info:max{name="other",state="disabled"} 8
cluster:console_plugins_info:max{name="other",state="notfound"} 9
```

Reply:
Just in theory yes

But this looks like a misconfiguration and is nothing we expect. Most clusters will have no plugin, or some from `redhat` or `other`, that are `enabled` or `disabled`.

Reply:
Ok so cardinality of 9 is the worst case but very unlikely. For most clusters, it should something between 0 and 4.

Reply:
Correct, I expect that most clusters will have `redhat` or `other` plugins in state `enabled` or `disabled`. So 0-4 is a reasonable assumption here.